Compile error with intel compiler

Compiling with intel 16.0.3 on EL7.2 and current git develop:
```
icc -O3 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -axSSE3,SSSE3,SSE4.1,SSE4.2,CORE-AVX2,COMMON-AVX512 -O2 -DMAX_STACK_ALLOC=2048 -wd981 -DF_INTERFACE_INTEL -fPIC -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=8 -DASMNAME= -DASMFNAME=_ -DNAME=_ -DCNAME= -DCHAR_NAME=\"_\" -DCHAR_CNAME=\"\" -DNO_AFFINITY -I. -O2 -DMAX_STACK_ALLOC=2048 -wd981 -DF_INTERFACE_INTEL -fPIC -DSMP_SERVER -DNO_WARMUP -DMAX_CPU_NUMBER=8 -DASMNAME= -DASMFNAME=_ -DNAME=_ -DCNAME= -DCHAR_NAME=\"_\" -DCHAR_CNAME=\"\" -DNO_AFFINITY -I..  -w -o linktest linktest.c ../libopenblas_nehalemp-r0.2.19.dev.so -L/opt/intel/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64 -L/opt/intel/compilers_and_libraries_2016.3.210/linux/ipp/lib/intel64 -L/opt/intel/compilers_and_libraries_2016.3.210/linux/mkl/lib/intel64 -L/opt/intel/compilers_and_libraries_2016.3.210/linux/tbb/lib/intel64/gcc4.4 -L/opt/intel/compilers_and_libraries_2016.3.210/linux/daal/lib/intel64_lin -L/opt/intel/compilers_and_libraries_2016.3.210/linux/tbb/lib/intel64_lin/gcc4.4 -L/opt/intel/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64_lin -L/opt/intel/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64_lin -L/usr/lib/gcc/x86_64-redhat-linux/4.8.5/ -L/usr/lib/gcc/x86_64-redhat-linux/4.8.5/../../../../lib64 -L/usr/lib/gcc/x86_64-redhat-linux/4.8.5/../../../../lib64/ -L/lib/../lib64 -L/lib/../lib64/ -L/usr/lib/../lib64 -L/usr/lib/../lib64/ -L/opt/intel/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64/ -L/opt/intel/compilers_and_libraries_2016.3.210/linux/mkl/lib/intel64/ -L/opt/intel/compilers_and_libraries_2016.3.210/linux/compiler/lib/intel64_lin/ -L/usr/lib/gcc/x86_64-redhat-linux/4.8.5/../../../ -L/lib64 -L/lib/ -L/usr/lib64 -L/usr/lib  -lifport -lifcore -limf -lsvml -lm -lipgo -lirc -lpthread -lsvml -lc -lirc_s -ldl -lc  -lgfortran && echo OK.
../libopenblas_nehalemp-r0.2.19.dev.so: undefined reference to `zerbla_'
make[1]: *** [../libopenblas_nehalemp-r0.2.19.dev.so] Error 1
make[1]: Leaving directory `/export/home/orion/fedora/openblas/OpenBLAS/exports'
make: *** [shared] Error 2
```
I see a reference to ZERBLA in lapack-netlib/SRC/zgetrf2.f, but no definition.

--- a/cpuid_x86.c
+++ b/cpuid_x86.c
@@ -1172,6 +1172,8 @@ int get_cpuname(void){
 #endif
           else
 	    return CPUTYPE_NEHALEM;
+	case 12:
+	  // Braswell
 	case 13:
 	  // Avoton
 	    return CPUTYPE_NEHALEM;

--- a/cpuid_x86.c
+++ b/cpuid_x86.c
@@ -1678,6 +1678,8 @@ int get_coretype(void){
 #endif
           else
 	    return CORE_NEHALEM;
+	case 12:
+	  // Braswell
 	case 13:
 	  // Avoton
 	    return CORE_NEHALEM;

--- a/driver/others/dynamic.c
+++ b/driver/others/dynamic.c
@@ -261,8 +261,8 @@ static gotoblas_t *get_coretype(void){
 	    return &gotoblas_NEHALEM; //OS doesn't support AVX. Use old kernels.
 	  }
 	}
-	//Intel Avoton
-	if (model == 13) { 
+	//Intel Braswell / Avoton
+	if (model == 12 || model == 13) { 
 	  openblas_warning(FALLBACK_VERBOSE, NEHALEM_FALLBACK); 
 	  return &gotoblas_NEHALEM;
 	}	

--- a/lapack-netlib/SRC/zgetrf2.f
+++ b/lapack-netlib/SRC/zgetrf2.f
@@ -144,7 +144,7 @@
       EXTERNAL           DLAMCH, IZAMAX
 *     ..
 *     .. External Subroutines ..
-      EXTERNAL           ZGEMM, ZSCAL, ZLASWP, ZTRSM, ZERBLA
+      EXTERNAL           ZGEMM, ZSCAL, ZLASWP, ZTRSM, XERBLA
 *     ..
 *     .. Intrinsic Functions ..
       INTRINSIC          MAX, MIN

